frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -125,6 +126,12 @@ const Settings: NextPage = () => {
     </li>
   ) : null;
 
+  const copyApiKeyToClipboard = (e: MouseEvent) => {
+    // This preventDefault was necessary to stop the form from submitting
+    e.preventDefault();
```

Comment:
That's because a `<button>` inside a `<form>` [is of type `submit` by default](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type), and hence clicking it submits the form.

(Of course the button shouldn't be in the form in the first place, UX-wise, but :shrug: :slightly_smiling_face:  )

Anyway, you can fix that by adding `type="button"` to the `<button>` element.